projects/risk-assessment-scores-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/RiskAssessmentService.kt

```diff
@@ -69,9 +76,21 @@ class RiskAssessmentService(
                 )
             )
             createContact(person, event, assessmentDate, ogrsScore)
+            createManagementTierEvent(person)
```

Comment:
Already approved but just noticed - do we need to do this for update as well?

Reply:
ha! yes